<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info_File</class>
 <widget class="QWidget" name="info_File">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Информация о Файле - </string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 47, 48);</string>
  </property>
  <widget class="QPushButton" name="ExitPushButton">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>173</y>
     <width>82</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_from_icon">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>78</width>
     <height>75</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="ImageLabel">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="FileNameLabel">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>90</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%FILE_NAME%</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>115</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип Файла:</string>
   </property>
  </widget>
  <widget class="QLabel" name="SizeLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>144</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Размер:</string>
   </property>
  </widget>
  <widget class="QLabel" name="FileTypeLabel">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>120</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%TYPE_FILE%</string>
   </property>
  </widget>
  <widget class="QLabel" name="FileSizeLabel">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>142</y>
     <width>132</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%SIZE_FILE%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="SelectFileSize">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>136</y>
     <width>65</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>KBs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MBs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GBs</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>92</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Имя файла:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="8" margin="8"/>
 <resources/>
 <connections>
  <connection>
   <sender>ExitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>info_File</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>false</bool>
  </property>
  <property name="gridSnapY">
   <bool>false</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
